packages/create-block-interactive-template/block-templates/render.php.mustache

```diff
@@ -1,33 +1,40 @@
 {{#isBasicVariant}}
 <?php
 /**
+ * PHP file to use when rendering the block type on the server to show on the front end.
+ *
+ * The following variables are exposed to the file:
+ *     $attributes (array): The block attributes.
+ *     $content (string): The block default content.
+ *     $block (WP_Block): The block instance.
+ *
  * @see https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-metadata.md#render
  */
 
 $unique_id = uniqid( 'p-' );
 ?>
 
 <div
-	<?php echo get_block_wrapper_attributes(); ?>
+	<?php echo wp_kses_data( get_block_wrapper_attributes() ); ?>
```

Comment:
Are you sure this is needed? I've never seen it before, and none of the Core blocks seems to do it.

Reply:
There is some differing opinions on this. WPCS is trigger because all `echo` statements need to be escaped. That being said, I see that the [default template in create-block](https://github.com/WordPress/gutenberg/blob/trunk/packages/create-block/lib/templates/block/render.php.mustache#L7) is not using it. So I'll remove it for consistency.